packages/common-ts/src/base-service/base-service-v2.ts

```diff
@@ -407,8 +435,6 @@ export abstract class BaseServiceV2<
 
       // Registery user routes.
       if (this.routes) {
-        const router = express.Router()
-        this.routes(router)
         app.use('/api', router)
```

Comment:
Can this whole block be moved up?

Reply:
The other block where I define router be move down. though.  Should I do that?